baseplate/lib/metrics.py

```diff
@@ -501,7 +503,7 @@ def metrics_client_from_config(raw_config: config.RawConfig) -> Client:
     """
     cfg = config.parse_config(
         raw_config,
-        {"metrics": {"namespace": config.String, "endpoint": config.Optional(config.Endpoint)}},
+        {"metrics": {"namespace": config.String, "endpoint": config.Optional(config.Endpoint), "histogram_sampling_rate": config.Optional(config.Float)}},
```

Comment:
Is it better to get the `histogram_sampling_rate` config option here in the same place where we get the other config options, then pass it through several constructors, or to pull it out of the raw_config later on when it's actually needed?